esmvaltool/diag_scripts/austral_jet/asr.ncl

```diff
@@ -180,7 +180,7 @@ begin
   ; ---------------------------------------------------------------------------
 
   ; Basic plotting settings
-  if (config_user_info@write_plots) then
+  if (diag_script_info@write_plots) then
```

Comment:
```suggestion
  if (.not. isatt(diag_script_info, "output_file_type")) then
    diag_script_info@write_plots = True
  end if
  if (diag_script_info@write_plots) then
```

Otherwise this will fail in version 2.4 when the `write_plots` attribute is not added to `diag_script_info` automatically anymore. Applies to all similar changes